packages/angular/src/migrations/update-8-3-0/upgrade-ngrx-8-0.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  chain,
+  Tree,
+  SchematicContext,
+  noop
+} from '@angular-devkit/schematics';
+import { addUpdateTask, readJsonInTree, formatFiles } from '@nrwl/workspace';
+import { ngrxVersion } from '../../utils/versions';
+
+const runNgrxUpdate = addUpdateTask('@ngrx/store', ngrxVersion);
```

Comment:
Let's use a hard coded version here in case people are migrating to not the latest version.